tasks/kyu7.py

```diff
@@ -0,0 +1,18 @@
+"""kyu7 Tasks"""
+
+from math import ceil
+
+
+def new_avg(arr, navg):
+    """Function find expected number that will permit to reach the average navg.
+        Args:
+            arr(list): array of numbers.
+            navg(int): expected average.
+        Returns:
+            Positive number or ValueError.
```

Comment:
Add "Raises" block to your function docstring

Reply:
I did. Thank you!